Bugzilla/Install/Localconfig.pm

```diff
@@ -40,10 +40,7 @@ our @EXPORT_OK = qw(
 
 sub _sensible_group {
     return '' if ON_WINDOWS;
-    my @groups     = qw( apache www-data _www );
```

Comment:
The default value is never used in current production, as localconfig is always present. A comment indicating that this is a divergence from upstream is probably needed though.